src/main/java/com/revature/services/TestService.java

```diff
@@ -0,0 +1,14 @@
+package com.revature.services;
+
+public class TestService {
+	
+	/*
+	public void breakBuild() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)